### DISCUSSION TITLE
【分类列表】文章分类显示“没有更多了”

### DISCUSSION THREAD

**Discussion by BazingLEE** [tangly1024/NotionNext#2057] [General]
**描述遇到的问题**
 #我在建立文章分类后，发现有时候点击进去可以看到分类的文章，但更多时候显示“没有更多了”。

**相应配置**
我的notion_page_id：56930b51df1e43dd8d4509e89fa054d5
我的网址：https://www.angi.fit/

**截图**
![image](https://user-images.githubusercontent.com/52346948/224634871-32d1da07-3574-4225-aef8-cc3e27a9eca3.png)

**环境**

- 操作系统: [例如.Android, windows]
- 浏览器 [例如. chrome, safari, firefox]
- 版本 [e.g. 22]

**Comment by kqfrv** [comment]
兄弟，你这个解决了

**Comment by kqfrv** [comment]
我也出现这个问题了 ，Notion 的文章没显示没显示

**Comment by BazingLEE** [comment]
还没解决呢。

**Comment by BazingLEE** [comment]
兄弟，我已经解决了。我试着在notion里删除了原来的分类，然后再次重新建立分类，再部署之后就可以了。
